latch/config/latch.py

```diff
@@ -4,7 +4,56 @@
 Platform wide configuration, eg. api endpoints, callback server ports...
 """
 
+DEV = "dev"
```

Comment:
I dont think we should release dev + staging urls with the public sdk, they are not designed to be used by non-internal developers